src/main/java/thePackmaster/cards/warlockpack/TheSoularium.java

```diff
@@ -39,6 +40,7 @@ public void use(AbstractPlayer p, AbstractMonster m) {
             public void update() {
                 for (AbstractCard c : DrawCardAction.drawnCards) {
                     CardModifierManager.addModifier(c, new EtherealMod());
+                    CardModifierManager.addModifier(c, new GlowTheSoulariumMod());
```

Comment:
I checked it out and this effect looks cool. Good reminder that the cards are Ethereal now (and need to be played this turn if you don't want them to exhaust).